packages/rest/lib/http.cadl

```diff
@@ -7,8 +7,8 @@ using Private;
 @doc("The request has succeeded.")
 model OkResponse<T> {
   @doc("The status code.")
-  @header
-  statusCode: 200;
+  @statusCode
+  _: 200;
```

Comment:
We said we would recommend against using `_` and still recommend a descriptive name. Please give these good names.